src/aggregate/expr/expression.c

```diff
@@ -46,21 +46,21 @@ static int evalFunc(ExprEval *eval, const RSFunctionExpr *f, RSValue *result) {
 
 static int evalOp(ExprEval *eval, const RSExprOp *op, RSValue *result) {
   RSValue l = RSVALUE_STATIC, r = RSVALUE_STATIC;
-  int rc = EXPR_EVAL_ERR;
 
   if (evalInternal(eval, op->left, &l) != EXPR_EVAL_OK) {
+    result = RS_NullVal();
     goto cleanup;
   }
   if (evalInternal(eval, op->right, &r) != EXPR_EVAL_OK) {
+    result = RS_NullVal();
     goto cleanup;
   }
 
   double n1, n2;
   if (!RSValue_ToNumber(&l, &n1) || !RSValue_ToNumber(&r, &n2)) {
-
-    QueryError_SetError(eval->err, QUERY_ENOTNUMERIC, NULL);
-    rc = EXPR_EVAL_ERR;
+    RSValue_SetNumber(result, NAN);
```

Comment:
Need to add label `pr:break` to the PR,
And to add to the PR description that an error will not be returned?